test/Table_Tests/src/Delimited_Read_Spec.enso

```diff
@@ -54,6 +55,69 @@ spec =
             r2 = File.read directory (File_Format.Delimited "," headers=True) Problem_Behavior.Report_Error
             r2.should_fail_with File.Io_Error
 
+        Test.specify "should work with all kinds of line endings" <|
+            path name = Enso_Project.data / 'transient' / name
+            create_file name ending_style =
+                lines = ['a,b,c', 'd,e,f', '1,2,3']
+                text = lines.join ending_style
+                (path name).write_text text
```

Comment:
should we explicitly write this in UTF_8 encoding
While I know logically the same will make it clearer here